packages/grafana-data/src/schema.ts

```diff
@@ -0,0 +1,8 @@
+// Moved to `@grafana/schema`, in Grafana 9, this will be removed
+//---------------------------------------------------------------
+// grafana/grafana/packages/grafana-schema$ grep export src/schema/*.ts
+
+export {
+  // Styles that changed
+  VizTextDisplayOptions as TextDisplayOptions, // rename
+} from '@grafana/schema';
```

Comment:
This looks good @ryantxu. But not all types will be renemed, right? Only the ones we are not happy with the name off. 

Talking dev experience now: I wonder if we can either have a linter for certain types imported from @grafana/data or deprecation notices on the re-exported ones. That would save a lot of work for ppl using IDE and automatic imports. 

I think this is an interesting alternative approach to allow deprecation notices: https://github.com/palantir/tslint/issues/3751 It's tslint related, but maybe will work with our setup? Worth checking out.

Reply:
Right, we only want to rename when that is an explicit choice.  This particular one was already in the code.  We had two copies of the same object so I just reflected that here :man_shrugging: